exist-core/src/main/java/org/exist/xquery/util/SerializerUtils.java

```diff
@@ -164,8 +191,9 @@ public enum ExistParameterConvention implements ParameterConvention<QName> {
         HIGHLIGHT_MATCHES("highlight-matches", Type.STRING, Cardinality.ZERO_OR_ONE, new StringValue("none")),
         JSONP("jsonp", Type.STRING, Cardinality.ZERO_OR_ONE, Sequence.EMPTY_SEQUENCE),
         ADD_EXIST_ID("add-exist-id", Type.STRING, Cardinality.ZERO_OR_ONE, new StringValue("none")),
-        ADD_NEWLINE_AT_EOF(EXistOutputKeys.INSERT_FINAL_NEWLINE, Type.BOOLEAN, Cardinality.ZERO_OR_ONE, BooleanValue.FALSE);
-
+        // default of 4 corresponds to the existing eXist default, although 3 is in the spec
+        INDENT_SPACES("indent-spaces", Type.INTEGER, Cardinality.ZERO_OR_ONE, new IntegerValue(4)),
+        INSERT_FINAL_NEWLINE(EXistOutputKeys.INSERT_FINAL_NEWLINE, Type.BOOLEAN, Cardinality.ZERO_OR_ONE, BooleanValue.FALSE);
```

Comment:
Why was `ADD_NEWLINE_AT_EOF` renamed to `INSERT_FINAL_NEWLINE`?

Reply:
It appears to be entirely unused. I think I decided that all the others had the same parameter name as their Java name, so I'd take the chance to make it consistent. If you'd prefer I reverse that, of course I can.

Reply:
That is a good reason to do so!